src/SchemaRegistry.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.21;
+
+import { Schema } from "./struct/Schema.sol";
+
+contract SchemaRegistry {
+  mapping(bytes32 id => Schema schema) public schemas;
```

Comment:
Please add comments (@dev, @notice, @... ) for:
* contracts & interfaces
* variables
* methods
* events
* errors
* struct
This should be documented in the WoW